regression/esbmc/github_578_success2/main.c

```diff
@@ -0,0 +1,10 @@
+#include <stdint.h>
+#include <assert.h>
+int main()
+{
+  int32_t x = 2;
+  x <<= 31;
```

Comment:
Is this really defined behaviour? I was under the impression that left-shifts touching "the sign bit" for any signed integer type are undefined.

Reply:
I checked via `-fsanitize=undefined`:
```
../regression/esbmc/github_578_success2/main.c:6:5: runtime error: left shift of 2 by 31 places cannot be represented in type 'int32_t' (aka 'int')
```
The verdict should be `VERIFICATION FAILED`.